.cirrus.yml

```diff
@@ -0,0 +1,5 @@
+container:
+  dockerfile: Dockerfile
```

Comment:
A container from `Dockerfile` will be pre-build and **cached** so it won't catch changes to your `Gemfile.lock`. Consider creating a separate Dockerfile for CI or maybe add `rspec` to your Dockerfile and enjoy [Zero-configuration Docker builds](https://medium.com/cirruslabs/zero-config-docker-builds-with-cirrus-ci-703db9c1e04e).

Reply:
Okay, I created separate Dockerfile and it seems to work. Thanks.

Reply:
Glad it worked for you! 🙌

Reply:
@fkorotkov It seems not to working, how can I set up Zero-configuration Docker builds for separate Dockerfile for CI?

Reply:
Fixed here https://github.com/seaki/rubicure-graphql/pull/37